analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -316,7 +316,7 @@ class Pub(
         val nameOfCurrentPackage = manifest["name"].textValue()
         val containsFlutter = "flutter" in dependencies
 
-        log.info { "buildDependencyTree for package $nameOfCurrentPackage " }
+        log.debug { "buildDependencyTree for package $nameOfCurrentPackage " }
```

Comment:
Unrelated, but would you mind to also improve the wording, like at least trimming the trailing space and adding a dot?